test/fail_compilation/reserved_version_switch.d

```diff
@@ -315,5 +316,6 @@ Error: version identifier `all` is reserved and cannot be set
 Error: version identifier `none` is reserved and cannot be set
 Error: version identifier `D_PreConditions` is reserved and cannot be set
 Error: version identifier `D_PostConditions` is reserved and cannot be set
+Error: version identifier `D_ProfileGC` is reserved and cannot be set
```

Comment:
The CI fails here. Did you rebase? If yes, the last change seems to be broken.

Reply:
Oops, missed one flag in the test case. Should work now.